Apigee/ManagementAPI/Developer.php

```diff
@@ -450,7 +450,7 @@ public function save($force_update = false, $old_email = null)
         // We must also do this when creating a developer ex nihilo in order
         // to set initial status. Otherwise new developer will have default
         // status, which is generally 'approved'.
-        $this->post($old_email . '?action=' . $cached_status);
+        $this->post($this->email . '?action=' . $cached_status);
```

Comment:
Can you tell me why this change was made? It makes it impossible to change the developer's email.